package kotlinx.coroutines;

import java.util.concurrent.CancellationException;

public final class d1 {
    public static final CancellationException a(String str, Throwable th2) {
        CancellationException cancellationException = new CancellationException(str);
        cancellationException.initCause(th2);
        return cancellationException;
    }
}
